spec/scrum_lint/runner_spec.rb

```diff
@@ -15,17 +20,40 @@ module ScrumLint
     it 'validates the selected board' do
       allow(Linter::MissingTaskList).to receive(:call).and_call_original
       allow(Linter::ExtraList).to receive(:call).and_call_original
+      allow(Linter::MissingTaskList).to receive(:call).and_call_original
       runner.([])
       expect(Linter::MissingTaskList).to have_received(:call)
         .with(instance_of(Board))
       expect(Linter::ExtraList).to have_received(:call).with(instance_of(Board))
+      expect(Linter::ExtraList).to have_received(:call).with(instance_of(Board))
     end
 
-    it 'checks contexts' do
-      allow(Linter::MissingContext).to receive(:call).and_call_original
-      runner.([])
-      expect(Linter::MissingContext).to have_received(:call)
-        .with(instance_of(Board))
+    describe 'card linters' do
+      it 'runs linters where card tags are superset of linter tags' do
+        allow(Linter::MissingHashTag).to receive(:call).and_call_original
+        allow(Linter::MissingContext).to receive(:call).and_call_original
+        allow(Linter::MissingExpendedPoints).to receive(:call)
+        runner.([])
+        expect(Linter::MissingHashTag).to have_received(:call)
+          .with(instance_of(Card))
+        expect(Linter::MissingContext).to have_received(:call)
+          .with(instance_of(Card))
+        expect(Linter::MissingExpendedPoints).not_to have_received(:call)
+      end
+
+      it 'runs linters where card tags are superset of linter tags' do
```

Comment:
can you update the it statement here to differentiate it from the one above?

Reply:
Yeah, good catch. Being half asleep is probably not the best time for programming. I'll also add a little more coverage.